modules/vaos/app/services/eps/draft_appointment_service/service_error.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Eps
+  class DraftAppointmentService
+    ##
+    # Error class for DraftAppointmentService errors
+    #
+    class ServiceError < StandardError
```

Comment:
You are missing a unit test for this class, then you can simplify the other tests as you do not need to test for content twice.